pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -180,12 +180,19 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 }
 
 func createMetaData(header []string) (addon.AddOnMeta, error) {
-	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
+	regex, _ := regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)
 	metaMap := make(map[string]string)
+	var key string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 && key != "" {
+			line = strings.TrimPrefix(line, commentChar)
+			metaMap[key] = metaMap[key] + " " + strings.TrimSpace(line)
```

Comment:
> Maybe we should even store the description as a slice of strings. That might be more flexible for various use cases. For example in the case of minishift addon list --verbose you would just display the first line, whereas an explicit minishift add-on describe <name> would write the full description content. Having a slice makes things a bit more flexible. Otherwise you might have to split yet again at newline.

We can store in the slice of strings but I didn't see any use until we break `createMetaData` function and have another function which return slice to different metadata key. Right now we are not enabling multi-line feature only for description but all sort of metadata tags.